### PR TITLE
Strung Rabbit Foot necklace

### PR DESCRIPTION
**Pull request by TastyPumPum**
Adds the crafting of the strung rabbit foot necklace. 

Adds the relevant effect onto the birdhouse and woodcutting activity, will have to be equipped for it to take place, as some people might prefer the seeds instead

-   [x] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/lib/skilling/skills/hunter/birdHouseTrapping.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export interface Birdhouse {
 	houseItemReq: Bank;
 	craftItemReq: Bank;
 	table: LootTable;
+	strungRabbitFootTable: LootTable;
```
Comment (TastyPumPum) [inline src/lib/skilling/skills/hunter/birdHouseTrapping.ts:17]: Theres probably an easier way to do this, any advice would be appreciated

**Comment by ColossalSeagull** [comment]
Does this affect birdhouses as well? Since it does affect egg nests on bh runs in game as well. 
image

**Comment by TastyPumPum** [comment]
Yes I mentioned that in my original post

File: src/lib/skilling/skills/crafting/craftables/misc.ts [hunk 2]
```diff
@@ -205,6 +205,14 @@ const Misc: Craftable[] = [
 		inputItems: new Bank({ Amethyst: 1 }),
 		tickRate: 2,
 		outputMultiple: 8
+	},
+	{
+		name: 'Strung rabbit foot',
+		id: itemID('Strung rabbit foot'),
+		level: 37,
+		xp: 4,
+		inputItems: new Bank({ 'Ball of wool': 1, 'Rabbit foot': 1 }),
```
Comment (gc) [inline src/lib/skilling/skills/crafting/craftables/misc.ts:214]: Switch to `.add`

File: src/tasks/minions/HunterActivity/birdhouseActivity.ts [hunk 3]
```diff
@@ -116,6 +121,10 @@ export const birdHouseTask: MinionTask = {
 
 			str += `\n\nYou received: ${loot}.`;
 
+			if (strungRabbitFoot) {
+				str += "\nYour strung rabbit foot necklace increases the chance of recieving bird's eggs and rings.";
```
Comment (gc) [inline src/tasks/minions/HunterActivity/birdhouseActivity.ts:125]: typo

File: src/tasks/minions/woodcuttingActivity.ts [hunk 4]
```diff
@@ -73,6 +76,10 @@ export const woodcuttingTask: MinionTask = {
 			str += `. **Bonus XP:** ${bonusXP.toLocaleString()}`;
 		}
 
+		if (strungRabbitFoot && !log.clueNestsOnly) {
+			str += "\nYour strung rabbit foot necklace increases the chance of recieving bird's eggs and rings.";
```
Comment (gc) [inline src/tasks/minions/woodcuttingActivity.ts:80]: typo

**Review by gc — CHANGES_REQUESTED** [review]

File: src/lib/skilling/skills/hunter/birdHouseTrapping.ts [hunk 5]
```diff
@@ -49,6 +54,10 @@ const birdhouses: Birdhouse[] = [
 			.every('Raw bird meat', [1, 5])
 			.tertiary(2, nestTable, [1, 2])
 			.tertiary(3, 'Feather', [10, 100]),
+		strungRabbitFootTable: new LootTable()
+			.every('Raw bird meat', [1, 5])
+			.tertiary(2, strungRabbitFootNestTable, [1, 2])
+			.tertiary(3, 'Feather', [10, 100]),
```
[the remark this answers is not in the document]
Comment (TastyPumPum) [reply src/lib/skilling/skills/hunter/birdHouseTrapping.ts]: Not sure if I've done it how you wanted it, but made some changes to it, let me know if it needs changing and if you could give further guidance would be appreciated.

**Pull request merged**